### PR TITLE
Add 'Bacterial genome annotation' workflow

### PR DESCRIPTION
**Pull request by clsiguret**
This workflow comes from the ABRomics project and can be used for any bacterial genomic analysis:
**Bacterial genome annotation**

This workflow uses 3 databases which must be available on http://datacache.galaxyproject.org/:
- [plasmidfinder-db](http://datacache.galaxyproject.org/indexes/plasmidfinder-db/) (**PlasmidFinder**): plasmidfinder_81c11f4_2023_12_04
- [bakta_database](http://datacache.galaxyproject.org/indexes/bakta_database/) (**Bakta**): V5.0_2023-02-20
- [amrfinderplus-db](http://datacache.galaxyproject.org/indexes/amrfinderplus-db/) (**Bakta**): amrfinderplus_V3.12_2024-05-02.2

We followed the recommendation to publish the workflow (licence, author list and email, steps annotations...)
- [x] Workflow imported from [usegalaxy.fr](https://usegalaxy.fr/) using `planemo workflow_test_init` to use a valid invocation
- [x] Workflow tests passed using `planemo test` command on the [usegalaxy.fr](https://usegalaxy.fr/) server
- [x] Readme.md written
- [x] CHANGELOG.md written
- [x] .dockstore.yml written

### CODE REVIEW

**Comment by clsiguret** [comment]
Hello :smile:

For PlasmidFinder, I use the `plasmidfinder_81c11f4_2023_12_04` database.
In the datacache [plasmidfinder-db](http://datacache.galaxyproject.org/indexes/plasmidfinder-db/), I already see that there is the database [81c11f4_2023_12_04/](http://datacache.galaxyproject.org/indexes/plasmidfinder-db/81c11f4_2023_12_04/).

Will it be the same and ok for the test here?

Thanks !

**Comment by mvdbeek** [comment]
I see this error in the logs:
```
parameter 'bakta_db_select': an invalid option ('V5.0light_2023-02-20') was selected (valid options: V5.0_2023-02-20)
```
can you use V5.0_2023-02-20 ? I'm not sure what the difference is

**Comment by clsiguret** [comment]
> I see this error in the logs:
> 
> ```
> parameter 'bakta_db_select': an invalid option ('V5.0light_2023-02-20') was selected (valid options: V5.0_2023-02-20)
> ```
> 
> can you use V5.0_2023-02-20 ? I'm not sure what the difference is

I juste chose this database. It's not a problem for me to change it to V5.0_2023-02-20 😊

**Review by mvdbeek — APPROVED** [review]
Very nice, thank you so much @clsiguret!

File: workflows/bacterial_genomics/bacterial_genome_annotation/.dockstore.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+workflows:
+- name: main
+  subclass: Galaxy
+  publish: true
+  primaryDescriptorPath: /bacterial_genome_annotation.ga
+  testParameterFiles:
+  - /bacterial_genome_annotation-tests.yml
+  authors:
+  - name: ABRomics
+    email: mailto:abromics-support@groupes.france-bioinformatique.fr
```
Comment (mvdbeek) [inline workflows/bacterial_genomics/bacterial_genome_annotation/.dockstore.yml:11]:
```suggestion
    email: abromics-support@groupes.france-bioinformatique.fr
```

**Pull request merged**